### PR TITLE
6185 build wsl page

### LINKED ISSUE #6185: /wsl: Build
Build ubuntu.com/wsl, as written in [Ubuntu-design#205](https://github.com/CanonicalLtd/Ubuntu-design/issues/205) and visually designed in #6184.

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added /wsl page, based on the [copy doc](https://docs.google.com/document/d/1M7e2B08GgPA7K6Wx0ThE6s9_xQINM0MSkarfdKUTEYI/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/wsl
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1M7e2B08GgPA7K6Wx0ThE6s9_xQINM0MSkarfdKUTEYI/edit#) and looks good

## Issue / Card

Fixes #6185

### CODE REVIEW

File: templates/wsl/index.html [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">In seconds, be able to access the linux terminal and run Linux applications and workflows on your Windows machine.</p>
```
Comment (pmahnke) [inline templates/wsl/index.html]: I think you pasted the paragraph in the heading...

File: templates/wsl/index.html [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL?action=subscribe#Installing_Packages_on_Ubuntu">Installing packages on Ubuntu on WSL&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

File: templates/wsl/index.html [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL#Running_Graphical_Applications">Running graphical applications with X on WSL&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

File: templates/wsl/index.html [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL?action=subscribe#Hello_Worlds">Hello World on WSL&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

File: templates/wsl/index.html [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL#Getting_Help">Getting help with Ubuntu on WSL&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

File: templates/wsl/index.html [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL#Performance">WSL performance tips&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

File: templates/wsl/index.html [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+              width="38",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Tutorial</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--four"><a class="p-link--external" href="https://wiki.ubuntu.com/WSL#Installing_Ubuntu_on_WSL_by_sideloading_the_.appx">Installing Ubuntu on WSL without Microsoft Store&nbsp;&rsaquo;</a></h3>
```
Comment (pmahnke) [inline templates/wsl/index.html]: remove the > from p-link--external

**Comment by sowasred2012** [comment]
@pmahnke those code issues are fixed, will keep an eye out for the social image and nav advice.

**Comment by pmahnke** [comment]
@sowasred2012 I added a few comments on the [Developer tab](https://docs.google.com/document/d/1JzZPBydcdyV96Hu4xRpa7VDU_WREr6gb05bL7GlQaYY/edit) and let's use this image for now for social: https://assets.ubuntu.com/v1/ad89548e-ubuntu-on-wsl.png

**Pull request merged**